### PR TITLE
AUTISM on Osmosis

### PR DESCRIPTION
**Pull request by PFC-developer**
I think i've got the right type 'ics20' here

### CODE REVIEW

File: osmosis/assetlist.json [hunk 1]
```diff
@@ -1169,6 +1169,53 @@
...
+            "channel_id": "channel-8",
+            "path": "transfer/channel-8/factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "injective",
+            "base_denom": "utia"
```
Comment (dylanschultzie) [inline osmosis/assetlist.json:1211]: This doesn't seem right?

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

File: osmosis/assetlist.json (hunk 1 above)
Comment (dylanschultzie) [reply osmosis/assetlist.json:1211]: @JeremyParish69 can you advise? I'm bad at these.

Comment (JeremyParish69) [reply osmosis/assetlist.json:1211]: It shouldn't be trying to synchronize to the TIA image. It should be pointing to the original AUTISM image. So base_denom is wrong here

Comment (PFC-developer) [reply osmosis/assetlist.json:1211]: I'll fix shortly.. does everything else  look ok?
I haven't done a token factory one before

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -1169,6 +1169,53 @@
...
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism",
+            "channel_id": "channel-122"
+          },
+          "chain": {
+            "channel_id": "channel-8",
+            "path": "transfer/channel-8/factory/inj14lf8xm6fcvlggpa7guxzjqwjmtr24gnvf56hvz/autism"
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:1203]: This path doesn't yield the ibc hash denom above when run through a sha256 function. But it does when using channel 122. Either the channels are backwards or the base hash is wrong

Comment (JeremyParish69) [reply osmosis/assetlist.json:1203]: yep, channels are backwards

File: osmosis/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply osmosis/assetlist.json:1211]: Other than my other comment, I think everything else is fine

File: osmosis/assetlist.json (hunk 2 above)
Comment (PFC-developer) [reply osmosis/assetlist.json]: thx. box changes have been done.

**Pull request merged**